Packs/GreatHorn/Integration/GreatHorn/GreatHorn.yml

```diff
@@ -0,0 +1,868 @@
+      description: ""
+      type: String
+    - contextPath: GreatHorn.Message.headers.x-ms-exchange-crosstenant-authas
+      description: ""
+      type: String
+    - contextPath: GreatHorn.Message.headers.x-ms-exchange-crosstenant-fromentityheader
+      description: ""
+      type: String
+    - contextPath: GreatHorn.Message.headers.x-ms-exchange-crosstenant-network-message-id
+      description: ""
+      type: String
+    - contextPath: GreatHorn.Message.headers.x-forefront-antispam-report
+      description: ""
+      type: String
+    - contextPath: GreatHorn.Message.headers.message-id
+      description: ""
+      type: String
+    - contextPath: GreatHorn.Message.headers.content-  type
+      description: ""
+      type: String
+    - contextPath: GreatHorn.Message.messageId
+      description: ""
+      type: String
+    - contextPath: GreatHorn.Message.authScore
+      description: ""
+      type: Number
+    - contextPath: GreatHorn.Message.headers.x-msfbl
+      description: ""
+      type: String
+    - contextPath: GreatHorn.Message.headers.content-transfer-encoding
+      description: ""
+      type: String
+    - contextPath: GreatHorn.Message.headers.precedence
+      description: ""
+      type: Unknown
+    - contextPath: GreatHorn.Message.headers.arc-seal
+      description: ""
+      type: Unknown
+    - contextPath: GreatHorn.Message.headers.arc-authentication-results
+      description: ""
+      type: Unknown
+    - contextPath: GreatHorn.Message.headers.arc-message-signature
+      description: ""
+      type: Unknown
+    - contextPath: GreatHorn.Message.headers.x-received
+      description: ""
+      type: Unknown
+    - contextPath: GreatHorn.Message.headers.to
+      description: ""
+      type: Unknown
+    - contextPath: GreatHorn.Message.headers.x-unsubscribe-web
+      description: ""
+      type: Unknown
+    - contextPath: GreatHorn.Message.headers.x-mailer
+      description: ""
+      type: Unknown
+    - contextPath: GreatHorn.Message.headers.x-info
+      description: ""
+      type: Unknown
+    - contextPath: GreatHorn.Message.headers.delivered-to
+      description: ""
+      type: Unknown
+    - contextPath: GreatHorn.Message.headers.mime-version
+      description: ""
+      type: Unknown
+    - contextPath: GreatHorn.Message.headers.x-tm-id
+      description: ""
+      type: Unknown
+    - contextPath: GreatHorn.Message.headers.x-rpcampaign
+      description: ""
+      type: Unknown
+    - contextPath: GreatHorn.Message.headers.x-google-smtp-source
+      description: ""
+      type: Unknown
+    - contextPath: GreatHorn.Message.headers.content-type
+      description: ""
+      type: Unknown
+  - name: gh-search-message
+    description: Search for message based on filtering input
+    arguments:
+    - name: fields
+      description: The fields to include in the response. By default, all fields are
+        returned.
```

Comment:
Please note: If a command uses argToList function the argument `isArray` should be set to true.

Reply:
done